### DISCUSSION TITLE
posts/2022/cloud-services-academia/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#32] [Blog comments]
# posts/2022/cloud-services-academia/

this is an experiment at making my Twitter conversations a bit more useful and archivable over time. It’s going to be a bit messy and unpolished, but hopefully that makes it more likely I’ll actual...

https://predictablynoisy.com/posts/2022/cloud-services-academia/

**Comment by robince** [comment]
I think one reason is objectively high costs (which is different from fear of overspending or budget overrun). For data intensive / memory bound (e.g. data preprocessing) compute costs are fine, but data ingress/egress and disk read/write costs make it prohibitively expensive compared to onprem. Cloud makes sense I think for more computational tasks (like training deep networks on a standard image dataset) but less for something like neuroimaging where you need to be streaming in data for preprocessing, writing large intermediate preprocessed data to disk etc. Compare the costs of hosted dedicated server (eg Scaleway) to cloud for data intensive workload. I understand the advantage of being able to rapidly scale, but that comes at a huge premium and is not so necessary in academic research.

**Comment by swnesbitt** [comment]
The university provides me free energy, networking, cooling, and IT support.  Is there a cost model for cloud that can beat that, especially for small to medium-sized computing problems?
